lib/ldap.js

```diff
@@ -28,7 +28,12 @@ function initializeConnection () {
   });
 
   connection.heartbeat = function (callback) {
-    connection.search('', '(objectclass=*)', function (err, res) {
+    var opts = {
+      scope: 'sub',
+      filter: nconf.get('LDAP_HEARTBEAT_SEARCH_QUERY'),
+      sizeLimit: 1
```

Comment:
please remove this, remember that it will return an error when query returns more than 1 entries

Reply:
Right, Thanks for pointing out. I will remove sizeLimit.